.github/workflows/build-release.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Just
+        run: sudo snap install --edge --classic just
+
+      - name: Build and Package
+        run: just zip
+
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: dist/*.zip
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't believe we need this, as it already defaults to the token.
[(also it's deprecated)
](https://github.com/softprops/action-gh-release?tab=readme-ov-file#environment-variables)
```suggestion
```